ex01.exs

```diff
@@ -28,8 +28,22 @@ defmodule Ex01 do
   """
   
   def counter(value \\ 0) do
+    receive do
```

Comment:
No deduction, but the accepted way of formatting this is 

~~~ elixir
{:next, from} -> 
  send from, {:next_is, value}
~~~